.homeycompose/capabilities/summer_winter.json

```diff
@@ -0,0 +1,9 @@
+{
+  "title": {
+    "en": "Season"
+  },
+  "type": "string",
+  "getable": true,
+  "setable": false,
+  "uiComponent": "sensor"
+}
```

Comment:
The sensors probably need a capability icon as well, otherwise they will appear with an empty placeholder in the UI.